src/features/rewards/tableContribute/index.tsx

```diff
@@ -133,11 +137,15 @@ export default class TableContribute extends React.PureComponent<Props, {}> {
       }
 
       if (this.props.showRemove) {
+        const actionTooltip = isExcluded
+          ? getLocale('restoreSite')
+          : getLocale('excludeSite')
+
         cell.content.push({
           content: (
-            <Tooltip content={getLocale('excludeSite')}>
+            <Tooltip content={actionTooltip}>
               <StyledRemove onClick={row.onRemove}>
-                <TrashOIcon />
+                <CloseStrokeIcon />
```

Comment:
we moved to trash icon, so this change is not needed

cc @rossmoody if this changed

Reply:
cc @jenn-rhim 
Should probably update this flow a little bit. I don't think we want to show a trash icon in the context of `restoring` a site. A green version of this might be better in the exclude panel: https://d.pr/i/luetMQ If that sounds ok, I can get that icon together.